packages/web3torrent/src/constants.ts

```diff
@@ -78,6 +73,15 @@ export const preSeededTorrents: Array<Partial<Torrent>> = [
   }
 ];
 
+// Welcome Page Tracker creation options
+export const welcomePageTrackerOpts = {
+  infoHash: [preSeededTorrents[0].infoHash],
+  announce: defaultTrackers,
+  peerId: '2d5757303030372d37454e613073307937495630', // random
```

Comment:
Should this be a randomly generated value instead of a hardcoded string? If there are many users using the Welcome page, will shared `peerId`s cause issues?

Reply:
Haven't tested that out, I _think_ it shouldn't make any problems, considering that this code doesn't actually download anything or makes any updates on the tracker

Reply:
I could make a random number every time though, so it doesn't become an issue